scss/base/_typography.scss

```diff
@@ -0,0 +1,28 @@
+html {
+  font-size: $base; /* set 1rem to value of $base */
+  line-height: $line-height-regular;
+}
+
+body {
+  color: $text-color;
+  font-family: $font-family-regular;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+h1 {
```

Comment:
These should be paired with a line-height value below `$base-lineheight` and set per variation. There should also be a size set for small screen (e.g. h1 48px/1.3 for large and h1 32px/1.3 for small)